lib/internal/validators.js

```diff
@@ -140,6 +141,10 @@ function validateBoolean(value, name) {
     throw new ERR_INVALID_ARG_TYPE(name, 'boolean', value);
 }
 
+function getOwnPropertyValueOrDefault(options, key, defaultValue) {
+  return options == null || !ObjectPrototypeHasOwnProperty(options, key) ? defaultValue : options[key];
```

Comment:
Could we use `Object.hasOwn` in node core now? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn

Reply:
It’s not available in primordials because it can be disabled with a runtime flag https://github.com/nodejs/node/blob/d96a2ea615b4090bb8cd53a9bc7fcba231479e12/deps/v8/src/flags/flag-definitions.h#L339

Reply:
Thanks for the info.